src/V3Const.cpp

```diff
@@ -2565,7 +3042,12 @@ class ConstVisitor final : public AstNVisitor {
         }
         // clang-format on
     }
-    virtual ~ConstVisitor() override = default;
+    virtual ~ConstVisitor() override {
+        if (m_doCpp && m_statBitOpReduction != 0) {
```

Comment:
```suggestion
        if (m_doCpp) {
```

Match other usages - seeing in stats an optimization had no effect is useful.